bin/setup

```diff
@@ -54,8 +54,11 @@ FileUtils.chdir APP_ROOT do
   system!("bundle check") || system!("bundle install")
 
   unless File.exist?('.env')
-    log "== Setup .env file from .env.example =="
+    log "== Setting up .env file from .env.example =="
     system!('cp .env.example .env')
+    puts  "Check .env to see if your database credentials are correct."
+    puts  "Press Enter to continue..."
+    gets
```

Comment:
I specifically did not use `log` because I think it is more confusing in yellow font.

Reply:
Can you put it in a red font maybe? Or blue? Something else that would stand out perhaps?

Reply:
![image](https://github.com/rubyforgood/human-essentials/assets/14540596/258d4326-cc02-4f32-9ac7-e05bd2821518)

That does look better.